quarkus/service/src/intTest/java/org/apache/polaris/service/quarkus/it/relational/jdbc/JdbcRestCatalogIT.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.quarkus.it.relational.jdbc;
+
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.quarkus.test.junit.TestProfile;
+import org.apache.polaris.service.it.test.PolarisRestCatalogIntegrationTest;
+import org.apache.polaris.test.commons.RelationalJdbcProfile;
+
+@TestProfile(RelationalJdbcProfile.class)
+@QuarkusIntegrationTest
+public class JdbcRestCatalogIT extends PolarisRestCatalogIntegrationTest {}
```

Comment:
There are three subclasses: 
1. JdbcRestCatalogIT
2. QuarkusRestCatalogIT
3. QuarkusRestCatalogIntegrationTest

Do we need all of them?

Reply:
I think only JdbcRestCatalogIT tests jdbc module E2E rest are all for in-memory persistence, i would prefer to still keep them as they helped in catch bug above in expected exception time.

Reply:
I believe we need integrations tests for the in-memory store as long as we use the in-memory store in practice (e.g. in the getting started guide).